### PR TITLE
ODIM-3207 add http handler for add/remove actions 

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: svc-api/handle/handlers.go [hunk 1]
```diff
@@ -965,3 +965,23 @@ func FabricsMethodNotAllowed(ctx iris.Context) {
 	fillMethodNotAllowedErrorResponse(ctx)
 	return
 }
+
+// AggregateMethodNotAllowed holds builds reponse for the unallowed http operation on Aggregate URLs and returns 405 error.
+func AggregateMethodNotAllowed(ctx iris.Context) {
+	url := ctx.Request().URL
+	path := url.Path
+	systemID := ctx.Params().Get("id")
```
Comment (jeevan-kamkar) [inline svc-api/handle/handlers.go]: rename it to aggregateID

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

File: svc-api/handle/handlers.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-api/handle/handlers.go]: updated

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

File: svc-api/handle/handlers_test.go [hunk 2]
```diff
@@ -490,3 +490,34 @@ func TestManagersMethodNotAllowed(t *testing.T) {
 	e.PATCH("/redfish/v1/Managers/{id}").Expect().Status(http.StatusMethodNotAllowed)
 	e.DELETE("/redfish/v1/Managers/{id}").Expect().Status(http.StatusMethodNotAllowed)
 }
+
```
Comment (Bharath-KKB) [inline svc-api/handle/handlers_test.go:493]: why are we adding test cases now and not before development?

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
why are we adding test cases now and not before development?

File: svc-api/handle/handlers_test.go (hunk 2 above)
Comment (irappa-pattar) [reply svc-api/handle/handlers_test.go:493]: this test case was missed in adding test cases PR

**Review by Bharath-KKB — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]
LGTM

**Pull request merged**